.github/workflows/main.yml

```diff
@@ -128,15 +128,56 @@ jobs:
 
   publish-flextesa-plugin-to-npm:
     needs:
-      - publish-ligo-plugin-to-npm
-      - publish-smartpy-plugin-to-npm
-      - publish-taquito-plugin-to-npm
+      - publish-sdk-to-npm
     uses: ecadlabs/taqueria/.github/workflows/npm-publish.yml@main
     with:
       dir: taqueria-plugin-flextesa
     secrets:
       NPM_TOKEN: ${{ secrets.NPMJS_PAT }}
 
+  run-tests:
+    needs: build-binaries
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./tests
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: get binary and make executable from previous step for tests
+        run: |
+          wget ${{ needs.build-binaries.outputs.public-url-ubuntu }} -O taq
+          chmod +x taq
+
+      - name: install dependencies for e2e tests in root and test directories
+        run: |
+          cd ..
+          curl -fsSL https://deno.land/x/install/install.sh | sh
+          npm ci
+          npm install --workspace=taqueria-plugin-ligo
+          npm install --workspace=taqueria-plugin-mock
+          
+          cd tests/
+          npm ci
```

Comment:
You could add `./tests` to the list of npm workspaces and change use
```
npm ci --workspace taqueria-plugin-ligo
npm ci --workspace taqueria-plugin-mock
npm ci --workspace test
```

Reply:
My ignorance of workspaces is showing. I will update it to include the tests folder as a workspace